src/components/editor/InputLiteral.jsx

```diff
@@ -125,6 +127,12 @@ export class InputLiteral extends Component {
     {
       id: idToRemove, label: labelToRemove
     })
+    this.props.formData.items.forEach(item => {
+      if(item.id === idToRemove) {
```

Comment:
Obviously not a big deal, but I notice this `if` statement has no space between the keyword and the opening paren, but the new `if` added on line 81 does have a space. Are we using a common linter for this codebase? Should we? Or are folks OK with variability in code style within the codebase?

As someone who's new to React (and not a Javascript expert generally), I'll largely learn by what I see, and the inconsistency can throw me off. That said, I can get over it if this isn't an area the team wants to care about at the moment. :smiley:

Reply:
Perhaps we should discuss at next retro?  We've been living with some whitespace inconsistency (but not indentation) in the js apps.   We were using at least 3 diff editors.